packages/trader/src/Modules/Contract/Containers/contract.jsx

```diff
@@ -28,7 +28,8 @@ class Contract extends React.Component {
                 {this.props.has_error ? (
                     <ErrorComponent
                         message={this.props.error_message}
-                        redirect_label={<Localize i18n_default_text='Go back to trading' />}
+                        is_dialog={this.props.error_code === 'ContractValidationError'}
+                        redirect_label={localize('Ok')}
```

Comment:
```suggestion
                        redirect_label={this.props.error_code === 'ContractValidationError' ? localize('Ok') : localize('Go back to trading')}
```

Reply:
for other non dialog errors we should still pass `Go back to trading` as button label